app/src/main/res/values/theme_naughtypenguins.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="NaughtyPenguins" parent="Theme.AppCompat.NoActionBar">
+        <item name="android:textColorPrimary">@android:color/holo_red_dark</item>
+        <item name="android:textColorSecondary">@android:color/darker_gray</item>
+        <item name="android:textColorTertiary">@android:color/holo_red_dark</item>
+        <item name="colorAccent">@android:color/holo_red_light</item>
```

Comment:
NaughtyPenguins was the very first theme created as a test. It's likely going to be scrapped in the final product. Credits to @forgeno for the naming inspiration btw. Good stuff